maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/StatelessXmlReporter.java

```diff
@@ -292,15 +293,15 @@ private FileOutputStream getOutputStream( WrappedReportEntry testSetReportEntry
 
         try
         {
-            return new FileOutputStream( reportFile );
+            return new BufferedOutputStream( new FileOutputStream( reportFile ), 16 * 1024 );
```

Comment:
I think we have problem.
`FileOutputStream outputStream = getOutputStream( testSetReportEntry );`
`OutputStreamWriter fw = getWriter( outputStream );`
Both variables are in use which is bad and the only one should be used.